ingeniamotion/enums/__init__.py

```diff
@@ -209,8 +212,3 @@ class FilterNumber(IntEnum):
 
     FILTER1 = 1
     FILTER2 = 2
-
-
-enums = list(globals().keys())
-enums.remove("IntEnum")
-__all__ = enums
```

Comment:
Beware that now the users that were doing
`from ingeniamotion.enums import *`

I don't see the problem of listing the enums that are public on the top of the file.
According to the PEP8 all these enums should be considered internal...
https://peps.python.org/pep-0008/#public-and-internal-interfaces

Reply:
Now we can only add a new enum and it will work fine. If we use __all__, we must add the enum to __all__, this is an unnecessary additional step.